mllib/src/main/scala/org/apache/spark/mllib/clustering/GaussianMixture.scala

```diff
@@ -187,7 +187,7 @@ class GaussianMixture private (
       case None =>
         val samples = breezeData.takeSample(withReplacement = true, k * nSamples, seed)
         (Array.fill(k)(1.0 / k), Array.tabulate(k) { i =>
-          val slice = samples.view.slice(i * nSamples, (i + 1) * nSamples)
+          val slice = samples.slice(i * nSamples, (i + 1) * nSamples)
```

Comment:
also cc @zhengruifeng could you help review the changes in mllib, thanks

Reply:
the ML part should be fine if all tests pass

Reply:
thanks @zhengruifeng